### PR TITLE
[Edit] Ruby:Comments

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

Revised the description to better align with our metadata standards, updated the entry to correct formatting errors, and finally added an example in addition to the existing syntax sections.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ *] All writings are my own.
- [* ] My entry follows the Codecademy Docs style guide.
- [* ] My changes generate no new warnings.
- [* ] I have performed a self-review of my own writing and code.
- [* ] I have checked my entry and corrected any misspellings.
- [ *] I have made corresponding changes to the documentation if needed.
- [* ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [* ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [* ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz Looks good to me.

File: content/ruby/concepts/comments/comments.md [hunk 1]
```diff
@@ -32,3 +32,21 @@ I am a multi line comment.
 I can take as many lines as needed.
 =end
 ```
+
+## Example
+
+The code below uses single-line comments.
```
Comment (yangc95) [inline content/ruby/concepts/comments/comments.md:38]:
```suggestion
The example below shows how single-line comments can be used in code:
```

File: content/ruby/concepts/comments/comments.md [hunk 2]
```diff
@@ -32,3 +32,21 @@ I am a multi line comment.
...
+
+```rb
+# This example uses a loop to count only the even values
+count = 0
+
+for i in 1..10 do
+  if (i % 2 == 0)
+    count += i
```
Comment (yangc95) [inline content/ruby/concepts/comments/comments.md:46]: given the comment above, I think this should be count += 1. Or line 41 can be changed to explain that the example will sum up the even values. I think it'd also be helpful to add some additional comments like when setting the variable `count` and then also looping through a range, etc

**Review by yangc95 — COMMENTED** [review]
@caupolicandiaz I had a few pieces of feedback! Tys for editing this entry 🙂

**Review by yangc95 — APPROVED** [review]
LGTM

**Pull request merged**